apps/api/src/main.ts

```diff
@@ -1,6 +1,6 @@
 import Fastify from 'fastify';
 import { app } from './app/app';
-import { logger } from './logger';
+import { logger } from '../../../libs/shared/src/logger';
```

Comment:
Nitpick: Would be great to add TS alias in the future

Reply:
I will do it now, cause I will forget